docs/jdbc-mysql-catalog.md

```diff
@@ -93,20 +98,20 @@ Please refer to [Manage Metadata Using Gravitino](./manage-metadata-using-gravit
 | `Binary`         | `Binary`    |
 
 :::info
-MySQL doesn't support Gravitino `Boolean` `Fixed` `Struct` `List` `Map` `Timestamp_tz` `IntervalDay` `IntervalYear` `Union` `UUID` type.
+MySQL doesn't support Gravitino `Boolean`, `Fixed`, `Struct`, `List`, `Map`, `Timestamp_tz`, `IntervalDay`, `IntervalYear`, `Union`, or `UUID` types.
 :::
 
 ### Table properties
 
-- Doesn't support table properties.
+The Gravitino table doesn't support table properties.
```

Comment:
I'd just put "Gravitino doesn't support table properties."

Reply:
More accurately, it should be stated that "Gravitino doesn't support table properties for the MySQL catalog." ?